apps/asctb-reporter/src/app/components/compare/compare.component.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { render } from '@testing-library/angular';
+
+import { of } from 'rxjs';
+import { CompareData } from '../../models/sheet.model';
+import { CompareComponent } from './compare.component';
+
+describe('CompareComponent', () => {
+  const VALID_SHEET_URL = 'https://docs.google.com/spreadsheets/d/1BxiMVs0XRA5nFMdKvBdBZjgmUUqptlbs74OgvE2upms/edit';
+
+  const createMockSheet = (overrides?: Partial<CompareData>): CompareData => ({
+    title: 'Test Sheet',
+    description: 'Test description',
+    link: `${VALID_SHEET_URL}#gid=123`,
+    color: '#FF0000',
+    fileName: 'test.csv',
+    formData: new FormData(),
+    sheetId: 'test-sheet-id',
+    gid: '123',
+    csvUrl: '',
+    ...overrides,
+  });
+
+  const renderComponent = async (compareSheets: CompareData[] = []) => {
+    const { fixture } = await render(CompareComponent, {
```

Comment:
Do not use `fixture` use https://testing-library.com/ queries and events to interact with the component/elements